pandas/tests/test_frame.py

```diff
@@ -11810,7 +11810,7 @@ def test_mode(self):
                            pd.DataFrame({"A": [12]}))
         assert_frame_equal(df[["D"]].mode(),
                            pd.DataFrame(pd.Series([], dtype="int64"),
-                                        columns=["D"]))
+                                        columns=["D"]), check_dtype=False)
```

Comment:
ok, create a separate issue for this specific case; this is related to ``Series.to_frame()``; point to this area in the code. We can do that for 0.17.0